other/docker/tokstyle/tokstyle.Dockerfile

```diff
@@ -15,8 +16,11 @@ RUN apt-get update && \
 COPY --from=tokstyle /bin/check-c /bin/
 RUN ["git", "clone", "--depth=1", "https://github.com/TokTok/hs-tokstyle", "/src/workspace/hs-tokstyle"]
 
-COPY toxav/ /src/workspace/c-toxcore/toxav/
-COPY toxcore/ /src/workspace/c-toxcore/toxcore/
-COPY toxencryptsave/ /src/workspace/c-toxcore/toxencryptsave/
-COPY third_party/cmp/cmp.h /src/workspace/c-toxcore/third_party/cmp/cmp.h
-RUN /bin/check-c $(find /src/workspace/c-toxcore -name "*.c")
+COPY --from=sources /src/ /src/workspace/c-toxcore/
+RUN /bin/check-c $(find /src/workspace/c-toxcore -name "*.c" \
+ -and -not -wholename "*/auto_tests/*" \
+ -and -not -wholename "*/other/*" \
```

Comment:
Doesn't that exclude the bootstrap daemon and other utilities when it didn't before? Do we not want to run check-c on them?

Reply:
It does, and [no we don't](https://github.com/TokTok/c-toxcore/blob/d6d67d56f3e56d30cefd81f70e68c041a8c4bf2a/other/docker/tokstyle/Dockerfile#L18-L21).